src/main/java/sg/ncl/testbed_interface/DataResource.java

```diff
@@ -14,5 +14,44 @@
 
     private Long id;
     private String uri;
+    private String maliciousFlag;
+    private boolean malicious;
+    private boolean scanned;
+
+
+    // override the malicious flag method
+    // to display the list of dropdown selection for admin to update malicious status of a data resource
+    // Note: values must be equivalent to those option values defined in the dropdown list under admin_data_resources_update.html
+    public String getMaliciousFlag() {
+        if (malicious & scanned) {
+            maliciousFlag = "true";
+        } else if (!malicious & scanned) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Correct this "&" to "&&". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2178)